### DISCUSSION TITLE
Are assigned IDs static or subject to change in the future?

### DISCUSSION THREAD

**Discussion by mainrs** [PokeAPI/pokeapi#752] [Q&A] [answered]
I want to incorporate the data provided by PokeAPI into a website of mine. However, I want the website to work offline as well (PWA). Thus, I decided to embed the needed data manually and not rely on the API itself.

I have to map the IDs of PokeAPI to the Pokemon index number used in the game save files. Are the assigned IDs of this project static? Can I be sure that once I create the mapping, it won't change?

**Comment by Naramsim** [comment, marked as answer]
No, you can't be sure. IDs are subject to change if new data gets pulled in.

**Reply by mainrs** [reply]
Is there any indicator that I can use to map in-game IDs to the PokeAPI IDs?

**Reply by Naramsim** [reply]
Hi, as general advice I wouldn't recommend hardcoding any IDs anywhere. That said, the probability that they will change for Pokeapi is pretty low.

**Comment by isikbattenstein** [comment]
Depending on what you would like to display, you can use the identifier instead. As far as I can see, the identifier wouldn't be that easy to reorganize compared to IDs. 
Change of ID is really low like @Naramsim says and I guess identifier would be lower.
@Naramsim , please correct me, if I am wrong.  :)
